rpcwebsocket.go

```diff
@@ -971,18 +971,18 @@ func (*wsNotificationManager) notifyNewTickets(clients map[chan struct{}]*wsClie
 	tnd *blockchain.TicketNotificationsData) {
 
 	// Create a ticket map to export as JSON.
-	var tickets []string
-	for _, h := range tnd.TicketsNew {
-		tickets = append(tickets, h.String())
+	ticketMap := make(map[string]string)
+	for i, ticket := range tnd.TicketsNew {
+		ticketMap[strconv.Itoa(i)] = ticket.String()
```

Comment:
That seemed silly to me too but I figured it wasn't worth changing this and winning tickets.  Winning tickets especially since It's already being used as-is in stakepoold.